src/io/sampling.jl

```diff
@@ -483,12 +413,45 @@ upper bound corresponds to the upper bound saved at the Domain's model_spec minu
 function get_bounds(dom::Domain, factor::Symbol)::Tuple
     model::Model = dom.model
     factor_filter::BitVector = collect(model[:fieldname]) .== factor
-    bounds::Tuple = model[:bounds][factor_filter][1]
+    bounds::Tuple = model[:dist_params][factor_filter][1]
 
     _is_discrete_factor(dom, factor) && return (bounds[1], bounds[2] - 1.0)
     return bounds
 end
 
+"""
+    get_bounds(param::Param)::Tuple
```

Comment:
Do (and why do) we need two separate docstrings for this function?

Reply:
I was thinking it's clearer with the arguments detailed separately, but you're right - consistency is better.